docs/methodology.md

```diff
@@ -1,48 +1,56 @@
 # Methodology
-Below we describe the methodology used to benchmark [CLP] and other tools.
+
+We benchmark ingestion and query performance for each tool. Since [CLP] supports both 
+**unstructured** and **dynamically-structured** logs, so we have separate benchmarks for each. 
+Below, we describe the datasets, benchmark machines, execution flow, and metrics collected.
```

Comment:
```suggestion
**unstructured** and **dynamically-structured** logs, we have separate benchmarks for each. Below,
we describe the datasets, benchmark machines, execution flow, and metrics collected.
```

Reply:
I think this "Below," cannot be moved to the end of the last line, otherwise it will exceed 100 wrapping restriction. (or the ',' is allowed to be the beginning of a newline)

Reply:
Yeah, you can apply the suggestion and then reflow to fit in 100 chars.